terraform/groups/lambda/profiles/development-eu-west-2/aard1/vars

```diff
@@ -9,3 +9,4 @@ aws_region = "eu-west-2"
 release_bucket_name = "development-eu-west-2.release.ch.gov.uk"
 remote_state_bucket = "ch-development-terraform-state-london"
 remote_state_key = "env:/development/development/development.tfstate"
+open_lambda_environment_variables = {LOGLEVEL="trace"}
```

Comment:
If LOGLEVEL="trace" is the same across a lot of environments, could it be set as default in variables.tf and then it could be overridden in the environment vars if need

Reply:
As per our chat, probably in this case is better to default it to an empty map and add entries where required rather than default it to "{LOGLEVEL="trace"}" and have to override it to an empty map for some cases. That might confuse folks tracing the logic in future.